<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>LoginUi</class>
 <widget class="QWidget" name="LoginUi">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>349</width>
    <height>400</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <property name="styleSheet">
   <string notr="true"/>
  </property>
  <widget class="QLineEdit" name="user_field">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>280</y>
     <width>270</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background: none; background-color: #ffffff;font-size:18pt; border-radius:2px; font-family:'Arial'; </string>
   </property>
  </widget>
  <widget class="QLineEdit" name="pass_field">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>320</y>
     <width>270</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background: none; background-color: #ffffff; font-size:18pt; border-radius:2px; font-family:'Arial';</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton">
   <property name="geometry">
    <rect>
     <x>120</x>
     <y>360</y>
     <width>118</width>
     <height>32</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton{background: none; background-color:#98B2BD; height: 5px; color: #ffffff; font-size:16pt; font-family:'Arial'; border-radius:2px;} QPushButton:hover{background-color:#798e97;}</string>
   </property>
   <property name="text">
    <string>Login</string>
   </property>
  </widget>
  <widget class="toolbar" name="widget_3" native="true">
   <property name="geometry">
    <rect>
     <x>-10</x>
     <y>0</y>
     <width>361</width>
     <height>51</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true"/>
   </property>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>908</x>
      <y>34</y>
      <width>131</width>
      <height>21</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>18</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">color:#6996AD;</string>
    </property>
    <property name="text">
     <string>Videoball</string>
    </property>
   </widget>
   <widget class="QPushButton" name="close_button_2">
    <property name="geometry">
     <rect>
      <x>1220</x>
      <y>0</y>
      <width>20</width>
      <height>23</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>14</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton:hover{color:#ffffff;}QPushButton{color:#7e7e7e;}</string>
    </property>
    <property name="text">
     <string>x</string>
    </property>
    <property name="flat">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QPushButton" name="min_button_2">
    <property name="geometry">
     <rect>
      <x>1197</x>
      <y>-6</y>
      <width>23</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>13</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton:hover{color:#ffffff;}QPushButton{color:#7e7e7e;}</string>
    </property>
    <property name="text">
     <string>_</string>
    </property>
    <property name="flat">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QPushButton" name="min_button">
    <property name="geometry">
     <rect>
      <x>315</x>
      <y>-5</y>
      <width>25</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>13</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton:hover{color:#ffffff;}QPushButton{color:#cccccc;}</string>
    </property>
    <property name="text">
     <string>_</string>
    </property>
    <property name="flat">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QPushButton" name="close_button">
    <property name="geometry">
     <rect>
      <x>336</x>
      <y>0</y>
      <width>21</width>
      <height>24</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>14</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton:hover{color:#ffffff;}QPushButton{color:#cccccc;}</string>
    </property>
    <property name="text">
     <string>x</string>
    </property>
    <property name="flat">
     <bool>true</bool>
    </property>
   </widget>
  </widget>
 </widget>
 <customwidgets>
  <customwidget>
   <class>toolbar</class>
   <extends>QWidget</extends>
   <header>toolbar.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
